### PR TITLE
V0.7.0 rc

### PR DESCRIPTION
**Pull request by bmhan12**
This PR:
- A release candidate
- The helper PR #917 has this branch target develop as the base to merge into for easier diffs
  -  **Adds comments and review to this PR for merging into main**, #917  is for viewing diffs.
- Manual Github Actions Windows build is passing: https://github.com/LLNL/axom/actions/runs/2920543574

### CODE REVIEW

File: RELEASE [hunk 1]
```diff
@@ -20,15 +20,16 @@ The Axom development team is comprised of:
...
-Esteban Pauli (pauli2@llnl.gov)
 Lee Taylor (taylor16@llnl.gov)
 Kenny Weiss (kweiss@llnl.gov)
 Chris White (white238@llnl.gov)
 Max Yang (wang39@llnl.gov)
 
 emeritus
 George Zagaris
+Josh Essman
+Pauli Esteban
```
Comment (kennyweiss) [inline RELEASE:35]:
```suggestion
Esteban Pauli
```

**Review by kennyweiss — APPROVED** [review]
Thanks @bmhan12 !

**Review by rhornung67 — COMMENTED** [review]
Overall looks great @bmhan12 Thank you. Please see my comments about citing all past contributors in https://github.com/LLNL/axom/pull/917

**Review by nselliott — APPROVED** [review]

**Review by agcapps — APPROVED** [review]

**Review by rhornung67 — APPROVED** [review]

**Comment by bmhan12** [comment]
Looking to merge into main, but there are these two required checks in Azure that are still waiting for status to be reported:
- LLNL.axom (Build_and_Test linux_gcc8)
- LLNL.axom (Build_and_Test linux_gcc8_64bit)

These might be from main? These jobs don't appear in the triggered Azure run, as the `azure-pipelines.yml` was upgraded to test gcc11 instead of gcc8.

**Comment by rhornung67** [comment]
@bmhan12 you are correct in that the azure builds for the main branch do gcc8, while develop (and this RC branch) do gcc11.  I don't know how to resolve this. I would think that the CI would run on a version of the code with this branch merged into main.

**Comment by kennyweiss** [comment]
> @bmhan12 you are correct in that the azure builds for the main branch do gcc8, while develop (and this RC branch) do gcc11. I don't know how to resolve this. I would think that the CI would run on a version of the code with this branch merged into main.

@white would probably know.
Perhaps we need a small patch PR into main to update the CI before merging in the RC branch?

**Comment by bmhan12** [comment]
I was trying to see if maybe starting a run through Azure Pipelines GUI instead of triggered by commit would solve the issue, but I don't have that access in Azure. Can someone grant me access?

**Comment by white238** [comment]
CI will run the jobs that are in the branch but Github protects the branch on specific jobs (which this branch is not running).  The way around this is to verify manually we are okay with merging, then have an authorized person turn off the specific branch protection for these tests, merge, then turn on the new ones for future PRs.

**Comment by rhornung67** [comment]
I don't think we should do a small PR to main to update the CI since that would pollute main, which should only contain merges that reflect the release history. Let's do what Chris suggested. @bmhan12 please verify that the RC branch works on a TOSS system with gcc8. Then, we can turn off the branch permissions on main and merge the PR.

**Comment by bmhan12** [comment]
>  @bmhan12 please verify that the RC branch works on a TOSS system with gcc8. Then, we can turn off the branch permissions on main and merge the PR.

Yes, as part of gitlab we test gcc 8 on ruby (TOSS3): https://lc.llnl.gov/gitlab/axom/axom/-/pipelines/203905

**Review by white238 — APPROVED** [review]

**Pull request merged**